tests/lib/core-config.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+#shellcheck source=tests/lib/systems.sh
+. "$TESTSLIB"/systems.sh
+
+clean_snapd_lib() {
+    rm -rf /var/lib/snapd/assertions/*
+    rm -rf /var/lib/snapd/device
+    rm -rf /var/lib/snapd/state.json
+}
+
+prepare_model(){
+    mv /var/lib/snapd/seed/assertions/model model.bak
+}
+
+restore_model(){
+    cp model.bak /var/lib/snapd/seed/assertions/model
+}
+
+prepare_seed(){
+    cp -f /var/lib/snapd/seed/seed.yaml seed.yaml.bak
```

Comment:
Why do we need `--force` here?

Reply:
I assume because on uc20 this file (seed.yaml) doesn't exist

Reply:
I don't get it, this is what `--force` is documented to do:
```
       -f, --force
              if an existing destination file cannot be opened, remove it  and
              try  again  (this  option  is ignored when the -n option is also
              used)
```

Reply:
I updated how the seed is prepared. Thanks for your review